app/models/labware_creators/pooled_tubes_by_submission.rb

```diff
@@ -28,7 +28,7 @@ def all_wells_in_pool_passed?(pool_info)
     end
 
     def pick?(location)
-      well_locations.fetch(location).passed?
+      !(well_locations.fetch(location).failed? || well_locations.fetch(location).empty?)
```

Comment:
Needed because when the multiplexing pipeline starts with a stock plate, the wells are of state 'unknown'